src/com/elixr/training/Main.java

```diff
@@ -0,0 +1,66 @@
+package com.elixr.training;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.StringTokenizer;
+import java.lang.String;
+
+public class Main {
+
+    public static void main(String[] args) {
+        if (args.length != 2) {
+            System.out.println("Argument missing"); //changed error message file path not valid to argument missing
```

Comment:
Please change this message to something like "File path and word to search are missing. Exiting the program... "

Reply:
message changed as mentioned above